corehq/apps/zapier/__init__.py

```diff
@@ -0,0 +1 @@
+default_app_config = 'corehq.apps.zapier.apps.ZapierConfig'
```

Comment:
you could just put this path in the settings file instead of `corehq.apps.zapier`